src/krux/input.py

```diff
@@ -42,35 +49,79 @@ def __init__(self):
         self.enter = GPIO(GPIO.GPIOHS21, GPIO.IN, GPIO.PULL_UP)
         fm.register(board.config["krux.pins"]["BUTTON_B"], fm.fpioa.GPIOHS22)
         self.page = GPIO(GPIO.GPIOHS22, GPIO.IN, GPIO.PULL_UP)
+        if "BUTTON_C" in board.config["krux.pins"]:
+            fm.register(board.config["krux.pins"]["BUTTON_C"], fm.fpioa.GPIOHS0)
+            self.previous = GPIO(GPIO.GPIOHS0, GPIO.IN, GPIO.PULL_UP)
+        self.has_touch = board.config["lcd"]["touch"]
+        self.touch = (
+            Touch(board.config["lcd"]["width"], board.config["lcd"]["height"], 50)
+            if self.has_touch
+            else None
+        )
+
+    def get_button_c(self):
```

Comment:
Two thoughts:
1) I think this pattern should be used for all buttons (i.e., `page` and `enter`)
2) The names of the methods ideally would match the underlying variable name, something like:  `page_state(self)`,  `enter_state(self)`, `previous_state(self)`, `touch_state(self)`

Also, try to avoid adding `get_` infront of the method names since it's not very "Pythonic". I've done it in a few places as well and need to change them...